common/authentication/postgresql/metadata.go

```diff
@@ -91,14 +91,10 @@ func (m *PostgresAuthMetadata) ValidateAwsIamFields() (string, string, string, e
 	if awsRegion == "" {
 		return "", "", "", errors.New("metadata property AWSRegion is missing")
 	}
+	// Note: access key and secret keys can be optional
```

Comment:
Does the metadata yaml need to be updated?

Reply:
no https://github.com/dapr/components-contrib/blob/main/state/postgresql/v2/metadata.yaml#L61

Reply:
I see, the yaml was previously wrong, now it's correct :laughing: